src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -868,6 +913,39 @@ def notifications_wrap_in_atomic_block(self):
             return False
         return super().notifications_wrap_in_atomic_block
 
+    def create(self, request, *args, **kwargs):
+        response = super().create(request, *args, **kwargs)
+        logger.info(
+            "zaakinformatieobject_created",
+            uuid=response.data.get("uuid"),
+            zaak_uuid=response.data.get("zaak"),
+            client_id=request.jwt_auth.client_id,
+        )
+        return response
+
+    def update(self, request, *args, **kwargs):
+        response = super().update(request, *args, **kwargs)
+        instance = self.get_object()
+        logger.info(
+            "zaakinformatieobject_updated",
+            uuid=str(instance.uuid),
+            zaak_uuid=str(instance.zaak.uuid) if instance.zaak else None,
+            client_id=request.jwt_auth.client_id,
+        )
+        return response
+
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        response = super().destroy(request, *args, **kwargs)
+        logger.info(
+            "zaakinformatieobject_deleted",
+            uuid=str(instance.uuid),
+            zaak_uuid=str(instance.zaak.uuid) if instance.zaak else None,
+            informatieobject_url=getattr(instance, "_objectinformatieobject_url", None),
+            client_id=request.jwt_auth.client_id,
+        )
+        return response
```

Comment:
Could you rewrite these with `perform_...`? That way it's consistent with the other viewsets